### ISSUE TITLE
Add ability to search a private pypi repo

### ISSUE THREAD

**Issue by taranlu-houzz** [frostming/oven#8]
Not sure if this is supported yet since this project is so new and I didn't see anything in the docs. It would be really cool if this could also be pointed at private pypi repos as well.

**Comment by frostming** [comment]
this relies on the /pypi/:name/json api but most private pypi repos don't provide that

**Comment by taranlu-houzz** [comment]
Sorry, I'm not super familiar with the web side of things. The project that I am most familiar with is this one: https://github.com/pypiserver/pypiserver, but I am not sure if it supports the api you are talking about...

Could this be it?
https://github.com/pypiserver/pypiserver#use-with-micropython

**Comment by taranlu-houzz** [comment]
Seems like perhaps this open issue is related: https://github.com/pypiserver/pypiserver/issues/508

And just to clarify, is this the api you are talking about: https://peps.python.org/pep-0691/

**Comment by frostming** [comment]
no. /pypi/json is never standardaized and that's why private repos don't support it

**Comment by taranlu-houzz** [comment]
Ah, I see. That's too bad then. Hopefully someday. Thanks for the clarification.
